api/datasets_test.go

```diff
@@ -20,183 +13,6 @@ import (
 	"github.com/qri-io/qri/lib"
 )
 
-func TestDatasetHandlers(t *testing.T) {
-	run := NewAPITestRunner(t)
-	defer run.Delete()
-
-	// Create a mock data server. Can't move this into the testRunner, because we need to
-	// ensure only this test is using the server's port "55555".
-	s := newMockDataServer(t)
-	defer s.Close()
-
-	h := NewDatasetHandlers(run.Inst, false)
-
-	// TODO: Remove this case, update API snapshot.
-	saveCases := []handlerTestCase{
-		{"POST", "/?new=true", mustFile(t, "testdata/newRequestFromURL.json"), nil},
-		{"DELETE", "/", nil, nil},
-	}
-	runHandlerTestCases(t, "init", h.SaveHandler(""), saveCases, true)
-
-	getCases := []handlerTestCase{
```

Comment:
The few `get` tests are useful for reference for a short while longer, but happy with them going away, more power to the new tests I guess. 👍

Reply:
Yea the issue here, was it felt like more work to get the tests working correctly, because they rely on the save handler saving datasets before "getting" the datasets, and the save handler doesn't exist anymore. Since we are planning a whole round of testing, it seemed like, exactly as you said, "more power to the new tests I guess".